components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpointTest.java

```diff
@@ -2137,6 +2164,162 @@ public Object answer(InvocationOnMock invocation) {
         assertEquals(cacheEntry[0].getoAuth2Parameters().getDisplayName(), savedDisplayName);
     }
 
+    @BeforeMethod
+    public void setupKeystore() throws Exception {
+
+        clientKeyStore = getKeyStoreFromFile("testkeystore.jks", "wso2carbon",
+                System.getProperty(CarbonBaseConstants.CARBON_HOME));
+    }
+
+    @DataProvider(name = "provideHandleRequestObjectData")
+    public Object[][] provideHandleRequestObjectData() throws Exception {
+
+        OAuth2Parameters oAuth2Parameters = new OAuth2Parameters();
+        oAuth2Parameters.setClientId(TestConstants.CLIENT_ID);
+        oAuth2Parameters.setRedirectURI(TestConstants.CALLBACK);
+        oAuth2Parameters.setTenantDomain(TestConstants.TENANT_DOMAIN);
+        oAuth2Parameters.setNonce("nonceInParams");
+        oAuth2Parameters.setState("stateInParams");
+        oAuth2Parameters.setPrompt("promptInParams");
+
+        Map<String, Object> defaultClaims = new HashMap<>();
+        defaultClaims.put(OAuthConstants.OAuth20Params.REDIRECT_URI, TestConstants.CALLBACK);
+        defaultClaims.put(NBF, System.currentTimeMillis() / 1000);
+        defaultClaims.put(EXP, System.currentTimeMillis() / 1000 + 3000);
```

Comment:
take 1000 and 3000 into constants.

Reply:
Fixed in d04102128137314d9c789e0d4a4bf9921b1bff01